examples/create_geopandas_from_pandas.py

```diff
@@ -0,0 +1,54 @@
+"""
+Creating a GeoDataFrame from a DataFrame
+----------------------------------------
+
+This example shows how to create a ``GeoDataFrame`` when starting from
+a *regular* ``DataFrame`` that has coordinates stored in two columns.
+
+"""
+import matplotlib.pyplot as plt
+import pandas as pd
+import geopandas as gpd
+from shapely.geometry import Point
+
+###############################################################################
+# First, let's consider a ``DataFrame`` containing cities and their respective
+# longitudes and latitudes.
+
+df = pd.DataFrame(
+    {'City': ['Buenos Aires', 'Brasilia', 'Santiago', 'Bogota', 'Caracas'],
+     'Country': ['Argentina', 'Brazil', 'Chile', 'Colombia', 'Venezuela'],
+     'Latitude': [-34.583333, -15.783333, -33.450000, 4.600000, 10.483333],
+     'Longitude': [-58.666667, -47.916667, -70.666667, -74.083333, -66.866667]})
+
+###############################################################################
+# A ``GeoDataFrame`` needs a ``shapely`` object, so we create a new column
+# **Coordinates** as a tuple of **Longitude** and **Latitude** :
+
+df['Coordinates']  = list(zip(df.Longitude, df.Latitude))
+
+###############################################################################
+# Then, we transform tuples to ``Point`` :
+
+df['Coordinates'] = df['Coordinates'].apply(Point)
```

Comment:
I was thinking here a bit about the best way how to do this. 

I think I previously sometimes did:

```
df.apply(lambda row: Point(row.Longitude, row.Latitude), axis=1)
```

but your version is definitely a lot faster, and I think also easier.